modules/adblocker/sources/config.es

```diff
@@ -120,6 +136,13 @@ class Config {
   set networkFetchEnabled(value) {
     prefs.set(ADB_FETCH, value);
   }
+
+  /**
+   * Readonly value.
+   */
+  get trustedSitesSnapshot() {
```

Comment:
Are the change related to "trusted sites" related to the injection change?

Reply:
Yes, because once onCommitted events are forwarded on trusted sites, then the adblocker is active (which it should not on trusted pages). Don't know the details, but in the existing code paths, everything was triggered through the webrequestAPI, and if Ghostery marked requests there as being handled, then the adblocker was skipped (I think).

But what I wanted to achieve that when you trust YouTube, for instance, then I would expect the adblocker not to be active on YouTube. And that decision needs now to be done in onCommitted as well. That is why ghostery-extension now pushes the trusted lists to common.